README.md

```diff
@@ -1,5 +1,11 @@
 # App token action
 
+> [!IMPORTANT]
+> This action is deprecated. Please migrate to the [github-app-token-action](https://github.com/actions/create-github-app-token). 
+>
+> Migration is almost straight forward, be-aware that GitHub actions requires the key as input and not the base64 encoded version of the private key.
```

Comment:
I would add a link to the documentation of actions/create-github-app-token on how to convert the base64 encoded key securely.

```suggestion
> Migration is almost straight forward, be-aware that GitHub actions requires the key as input and not the base64 encoded version of the private key. See [here](https://github.com/actions/create-github-app-token?tab=readme-ov-file#private-key) how to do that securely.
```